### PR TITLE
Distinguish between internal and external APIs

### PR DESCRIPTION
**Pull request by tfrauenstein**
And make API Discovery support mandatory only for external APIs.

### CODE REVIEW

File: api-operation/ApiOperation.md [hunk 1]
```diff
@@ -1,26 +1,33 @@
...
 * “Twintip” discovery endpoint `https://example.com/.well-known/schema-discovery` that delivers
   the OpenAPI definition endpoint(s) above (see the link below for a description of its format).
 
-Note, these discovery endpoints have to be supported but need not be part of the OpenAPI definition as there is no API specific information in their description.
+Hint: Though discovery endpoints have to be supported, they should not be specified in the OpenAPI definition 
+as they are generic and provide no API specific information.
 
-Background: [Twintip](http://docs.stups.io/en/latest/components/twintip.html) is an API definition
-crawler of the STUPS infrastructure; it checks all running applications via the endpoint above and
+We differ between internal and external APIs of an application which is owned by a specific team and often
```
Comment (ePaul) [inline api-operation/ApiOperation.md]: Editorial: I think "differ" is the wrong word here. You want "differentiate" or "distinguish".

(The same mistake is already in master in Compatibility.md and ProprietaryHeaders.md.)

File: api-operation/ApiOperation.md [hunk 2]
```diff
@@ -1,26 +1,33 @@
...
 
-Note, these discovery endpoints have to be supported but need not be part of the OpenAPI definition as there is no API specific information in their description.
+Hint: Though discovery endpoints have to be supported, they should not be specified in the OpenAPI definition 
+as they are generic and provide no API specific information.
 
-Background: [Twintip](http://docs.stups.io/en/latest/components/twintip.html) is an API definition
-crawler of the STUPS infrastructure; it checks all running applications via the endpoint above and
+We differ between internal and external APIs of an application which is owned by a specific team and often 
+implemented via small set of services. An external API is used by clients outside the team - usually an 
+other application owned by a different team or even an external business partner user of our platform.
```
Comment (ePaul) [inline api-operation/ApiOperation.md]: "an other" → "another"

File: api-operation/ApiOperation.md [hunk 3]
```diff
@@ -1,26 +1,33 @@
 # API Operation
 
-## {{ book.must }} Applications Must Provide Online Access to Their API (Swagger) Definitions
+## {{ book.must }} Provide Online Access to OpenAPI Reference Definition
 
-In our dynamic and complex service infrastructure, it is important to provide a central place with
-online access to the API definitions of all running applications. All service applications must
-provide the following two API endpoints:
+All service applications must online provide the OpenAPI Reference Definition for their external APIs — it 
+is optional for internal APIs — via the following two API endpoints:
```
Comment (ePaul) [inline api-operation/ApiOperation.md]: Editorial: The "online" doesn't fit at this location in the sentence. I would put it before "via".

File: api-operation/ApiOperation.md (hunk 2 above)
Comment (tfrauenstein) [inline api-operation/ApiOperation.md]: done.

File: api-operation/ApiOperation.md (hunk 3 above)
Comment (tfrauenstein) [inline api-operation/ApiOperation.md]: ok, have changed it:
All service applications must support access to the OpenAPI Reference Definitions of their external APIs...

**Comment by tlossen** [comment]
👍

**Comment by dehora** [comment]
👍

**Pull request merged**